src/olympia/bandwagon/tests/test_views.py

```diff
@@ -293,9 +292,9 @@ def test_publisher(self):
         CollectionUser.objects.create(collection=self.c, user=u)
         self.client.login(username='fligtar@gmail.com', password='foo')
         r = self.client.get(self.url)
-        eq_(r.status_code, 200)
+        assert r.status_code == 200
         # TODO(cvan): Uncomment when bug 719512 gets fixed.
-        #eq_(pq(r.content)('.meta .view-stats').length, 1,
+        # assert pq(r.content)('.meta .view-stats').length == 1, (
```

Comment:
Leaving this as TODO makes me sad as bug 719512 was fixed years ago but I don't know enough about the collections pages to fix (.meta .view-stats is always empty)